+++
note = "cited By 47"
doi = "10.1186/1471-2105-9-181"
url = "/publication/2008-pcm-hiv-protease"
title = "Proteochemometric modeling of HIV protease susceptibility"
url_html = "https://www.scopus.com/inward/record.uri?eid=2-s2.0-42949131067&partnerID=40&md5=42a9e17b56ed854787ce404f21aa0e5f"
journal = "BMC Bioinformatics"
author = "Lapins, M. and Eklund, M. and Spjuth, O. and Prusis, P. and Wikberg, J.E.S."
volume = "9"
source = "Scopus"
art_number = "181"
year = "2008"
date = "2008-04-10"
keywords = "Candidate drugs;  Drug research;  Drug resistance;  HIV-protease;  Physicochemical property;  Predictive abilities;  Protease inhibitor;  Treatment selection, Viruses, Diseases, amprenavir;  atazanavir;  indinavir;  lopinavir;  nelfinavir;  proteinase;  proteinase inhibitor;  ritonavir;  saquinavir;  proteinase, antiviral susceptibility;  article;  chemometrics;  controlled study;  correlation analysis;  genetic variability;  genotype;  Human immunodeficiency virus;  mutational analysis;  nonhuman;  physical chemistry;  prediction;  protein database;  virus morphology;  virus mutation;  antiviral resistance;  binding site;  chemical model;  chemistry;  combinatorial chemistry;  computer simulation;  drug delivery system;  methodology;  protein analysis;  protein binding, Binding Sites;  Combinatorial Chemistry Techniques;  Computer Simulation;  Drug Delivery Systems;  Drug Resistance, Viral;  HIV Protease;  HIV Protease Inhibitors;  Models, Chemical;  Protein Binding;  Protein Interaction Mapping"
document_bibtex_type = "Article"
abstract = "Background: A major obstacle in treatment of HIV is the ability of the virus to mutate rapidly into drug-resistant variants. A method for predicting the susceptibility of mutated HIV strains to antiviral agents would provide substantial clinical benefit as well as facilitate the development of new candidate drugs. Therefore, we used proteochemometrics to model the susceptibility of HIV to protease inhibitors in current use, utilizing descriptions of the physico-chemical properties of mutated HIV proteases and 3D structural property descriptions for the protease inhibitors. The descriptions were correlated to the susceptibility data of 828 unique HIV protease variants for seven protease inhibitors in current use; the data set comprised 4792 protease-inhibitor combinations. Results: The model provided excellent predictability (R2 = 0.92, Q2 = 0.87) and identified general and specific features of drug resistance. The model's predictive ability was verified by external prediction in which the susceptibilities to each one of the seven inhibitors were omitted from the data set, one inhibitor at a time, and the data for the six remaining compounds were used to create new models. This analysis showed that the over all predictive ability for the omitted inhibitors was Q2 inhibitors = 0.72. Conclusion: Our results show that a proteochemometric approach can provide generalized susceptibility predictions for new inhibitors. Our proteochemometric model can directly analyze inhibitor-protease interactions and facilitate treatment selection based on viral genotype. The model is available for public use, and is located at HIV Drug Research Centre. \u00a9 2008 Lapins et al; licensee BioMed Central Ltd."
+++

