+++
publisher = "Springer Nature"
doi = "10.1186/2047-217x-2-9"
author = "Lampa S, Dahl√∂ M, Olason PI, Hagberg J, Spjuth O"
url_html = "http://dx.doi.org/10.1186/2047-217x-2-9"
journal = "GigaScience"
title = "Lessons learned from implementing a national infrastructure in Sweden for storage and analysis of next-generation sequencing data"
number = "1"
month = "jun"
volume = "2"
year = "2013"
date = "2013-06-25"
bibtex_type = "article"
abstract = "Analyzing and storing data and results from next-generation sequencing (NGS) experiments is a challenging task, hampered by ever-increasing data volumes and frequent updates of analysis methods and tools. Storage and computation have grown beyond the capacity of personal computers and there is a need for suitable e-infrastructures for processing. Here we describe UPPNEX, an implementation of such an infrastructure, tailored to the needs of data storage and analysis of NGS data in Sweden serving various labs and multiple instruments from the major sequencing technology platforms. UPPNEX comprises resources for high-performance computing, large-scale and high-availability storage, an extensive bioinformatics software suite, up-to-date reference genomes and annotations, a support function with system and application experts as well as a web portal and support ticket system. UPPNEX applications are numerous and diverse, and include whole genome-, de novo- and exome sequencing, targeted resequencing, SNP discovery, RNASeq, and methylation analysis. There are over 300 projects that utilize UPPNEX and include large undertakings such as the sequencing of the flycatcher and Norwegian spruce. We describe the strategic decisions made when investing in hardware, setting up maintenance and support, allocating resources, and illustrate major challenges such as managing data growth. We conclude with summarizing our experiences and observations with UPPNEX to date, providing insights into the successful and less successful decisions made."
+++

